cli/index.adoc

```diff
@@ -233,23 +233,51 @@ This step is not Skupper-specific, that is, this process is unchanged from stand
 . Create a service that can communicate on the {service-network}:
 +
 --
+
+.Deployments and pods
 [source, bash]
 ----
-$ skupper expose [deployment <name>|pods <selector>|statefulset <statefulsetname>|service <name>]
+$ skupper expose [deployment <name>|pods <selector>]
 ----
 
 where 
 
-* `<name>` is the name of your deployment 
+* `<name>` is the name of a deployment 
 * `<selector>` is a pod selector
+
+.Kubernetes services
+Specify a resulting service name using the `--address` option.
+[source, bash]
+----
+$ skupper expose service <name> --address <skupper-service-name>
+----
+
+where 
+
+* `<name>` is the name of a service
+* `<skupper-service-name>` is the name of the resulting service shared on the {service-network}.
+
+.StatefulSets
+
+A StatefulSet in Kubernetes is typically associated with a headless service to provide stable, unique network identifiers for each pod.
+Use the `--headless` option to allow StatefulSets be exposed using Skupper.
```

Comment:
This makes it sound like you have to use --headless if the target is a statefulset. Perhaps just intiially show the basic form for a statefulset, then make the point about headless services and how to expose stable pod identifiers through a skupper network?

Reply:
yes, I was trying to introduce statefulset as 'typically associated with a headless service', but pushed the idea too far. I've rewritten to make it clearer,  optional `[--headless]` while avoiding expanding the text, wdyt?

Reply:
Though better, I think it still pushes the headless option too soon and possibly for cases where it is not required. I think it would be better to describe the behaviour/feature that might be needed, e.g. something like, "if you need to be able to address the pods in this statefulset individually from other sites, you can use the --headless option". We should also point out that in this case only the local statefulset can be exposed through the same service. (Whereas if you don't use --headless, then you could expose deployments and statefulsets from any site in the network through the same address).

Reply:
@grs Lemme see if I understand

1. If site A and B both have sts named test, and in context of site A:
`skupper expose statefulset test` 
Then a call from site C could resolve to site B?

Would it be simpler for user to create a service from the test sts and expose that?

 2. If I want to call a specific pod, I could just expose the pod, but I'd have to name each service. 
 With  `skupper expose statefulset test --headless` i get each pod exposed, and the services will change as the number in the statefulset is scaled up and down 
 
 does that sound right?

Reply:
@grs Ignore my last comment, I've updated based on your feedback. Note `another site using the same address (routing key)`, is that correct? I plan to use the following convention on the 1.x stream docs so that we're clear when using the word address:

* address (service name)
* address (routing key)